docs/tutorials/07_AE_workflow.ipynb

```diff
@@ -0,0 +1,2159 @@
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>301.720689</td>\n",
+       "      <td>0.441374</td>\n",
+       "      <td>592.077527</td>\n",
+       "      <td>3.211052</td>\n",
+       "      <td>12.050278</td>\n",
+       "      <td>94.784349</td>\n",
+       "      <td>3.349951</td>\n",
+       "      <td>2.992169</td>\n",
+       "      <td>0.437612</td>\n",
+       "      <td>0.660395</td>\n",
+       "      <td>13.133511</td>\n",
+       "      <td>0.071596</td>\n",
+       "      <td>1.260190</td>\n",
+       "      <td>1.894247</td>\n",
+       "      <td>13.682206</td>\n",
+       "      <td>0.034197</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         ao.r      ao.c        art.r     art.c      ven.r       ven.c  \\\n",
+       "0  264.203974  0.166940  1469.862481  1.856207  10.168458   81.346074   \n",
+       "1  205.666326  0.298874   795.708507  3.595813   4.501636  147.850535   \n",
+       "2  256.892858  0.230038  1382.506725  2.759971   9.885508  196.564487   \n",
+       "3  127.848640  0.417055  1074.358099  3.655236  11.111040  106.884289   \n",
+       "4  301.720689  0.441374   592.077527  3.211052  12.050278   94.784349   \n",
+       "\n",
+       "       av.r      mv.r  la.E_pas  la.E_act   la.v_ref  la.k_pas  lv.E_pas  \\\n",
+       "0  8.227971  3.625817  0.567897  0.318146   5.422773  0.043633  1.498943   \n",
+       "1  7.697389  4.241722  0.616009  0.622878   6.259190  0.051676  0.583826   \n",
+       "2  5.933286  3.831553  0.278264  0.291852  11.006724  0.032594  1.356216   \n",
+       "3  4.024437  4.854296  0.597769  0.268592   8.173386  0.047343  0.689071   \n",
+       "4  3.349951  2.992169  0.437612  0.660395  13.133511  0.071596  1.260190   \n",
+       "\n",
+       "   lv.E_act   lv.v_ref  lv.k_pas  \n",
+       "0  2.925541   9.323531  0.035199  \n",
+       "1  1.701070   8.058906  0.044577  \n",
+       "2  4.136944  14.241581  0.014962  \n",
+       "3  2.503425  11.457123  0.023844  \n",
+       "4  1.894247  13.682206  0.034197  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from autoemulate.experimental_design import LatinHypercube\n",
+    "\n",
+    "# Generate Latin Hypercube samples\n",
+    "N_samples = 10\n",
+    "lhd = LatinHypercube(list(parameters_range.values()))\n",
+    "sample_array = lhd.sample(N_samples)\n",
+    "sample_df = pd.DataFrame(sample_array, columns=parameters_range.keys())\n",
+    "\n",
+    "print(\"Number of parameters:\", sample_df.shape[1], \"Number of samples from each parameter:\", sample_df.shape[0])\n",
+    "sample_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3 - Wrapping your Simulator in AutoEmulate Simulator Base Class\n",
```

Comment:
To make this as straightforward for the user as possible, we could move the `sample_inputs()` method implementation from Naghavi to the Base Simulator. A user could always choose to override it but in most instances we want to do LatinHypercube sampling so it seems like a good default for all simulators to inherit. 

If we do that, then we could simplify this section and simply say, the user has to subclass the Simulator class and implement the `sample_forward` method (I wouldn't even worry about mentioning `_init_` here). I think that makes it really accessible.